packages/bundler/provider.ts

```
@@ -109,19 +126,23 @@ export class BundlerProvider {
       blockNumber: BigNumberish
     }
   }> {
-    const receipt = await this.bundler.send<{
-      success: boolean
-      reason: string
-      receipt: {
-        transactionHash: HexString
-        blockHash: HexString
-        blockNumber: BigNumberish
-      }
-    }>({
-      method: BundlerRpcMethod.eth_getUserOperationReceipt,
-      params: [userOpHash]
-    })
-    return receipt
+    try {
+      const receipt = await this.bundler.send<{
+        success: boolean
+        reason: string
+        receipt: {
+          transactionHash: HexString
+          blockHash: HexString
+          blockNumber: BigNumberish
+        }
+      }>({
+        method: BundlerRpcMethod.eth_getUserOperationReceipt,
+        params: [userOpHash]
+      })
+      return receipt
+    } catch (err) {
+      console.log("getUserOperationReceipt error", err)
```

Only logging without propagation the error might be hidden and unaware.